modules/api/pkg/plugin/detail_test.go

```diff
@@ -41,7 +41,7 @@ func TestGetPluginSource(t *testing.T) {
 	pcs := fakePluginClientset.NewSimpleClientset()
 	cs := fakeK8sClient.NewSimpleClientset()
 
-	_, err := GetPluginSource(pcs, cs, ns, pluginName)
+	_, err := GetPluginSource(context.TODO(), pcs, cs, ns, pluginName)
```

Comment:
Use `context.Background()` instead in this file.

Reply:
@maciaszczykm I replaced `context.TODO()` with `context.Background()` in the test.

In the issue it was also mentioned that there are more instances of `context.TODO()`. I searched the files and in the plugin package there is also `GetPluginList` that uses `context.TODO()`. Should I replace that one too?